applications/FluidDynamicsApplication/python_scripts/vms_fractional_step_solver.py

```diff
@@ -4,6 +4,19 @@
 from KratosMultiphysics import *
 from KratosMultiphysics.FluidDynamicsApplication import *
 
+import KratosMultiphysics.python_linear_solver_factory as linear_solver_factory
+
+try:
+    import KratosMultiphysics.MeshingApplication as KMesh
+except:
+    raise("MeshingApplication is missing")
+
+try:
+    import KratosMultiphysics.IncompressibleFluidApplication 
+    from KratosMultiphysics.IncompressibleFluidApplication import new_restart_utilities
+except:
+    raise("IncompressibleFluidApplication is missing")
+
```

Comment:
I also put wards here, notice that in this case as the imports were previously inside functions I am extending the restriction to the whole script, which I am not sure if its correct or not, but if it is not, I think the solution would be to split the script and separate the functions that have dependencies rather than maintaning the import in the middle.